pkg/cmd/util/cmd.go

```diff
@@ -95,6 +95,18 @@ func convertItemsForDisplay(objs []runtime.Object, preferredVersions ...schema.G
 			groups = append(groups, groupMeta)
 		}
 
+		// if no preferred versions given, convert item to first group version found.
+		// the given object is guaranteed to be at least part of one group by this point.
+		if len(preferredVersions) == 0 {
+			defaultGroupVersion := groups[0].GroupVersion
+			convertedObject, err := kapi.Scheme.ConvertToVersion(obj, defaultGroupVersion)
```

Comment:
You should just be passing in groupVersions to ConvertToVersion (you'll need to cast to the runtime schema type which handles Versioner).